soc/nxp/imx/imx9/a55/mmu_regions.c

```diff
@@ -38,6 +38,9 @@ static const struct arm_mmu_region mmu_regions[] = {
 	MMU_REGION_DT_COMPAT_FOREACH_FLAT_ENTRY(nxp_kinetis_lpuart,
 			 (MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS))
 
+	MMU_REGION_DT_COMPAT_FOREACH_FLAT_ENTRY(nxp_flexcan,
+			 (MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS))
+
```

Comment:
Would you please change the driver can_mcux_flexcan.c to use MMIO mapping (refer to https://docs.zephyrproject.org/latest/kernel/drivers/index.html#memory-mapping), so that we don't need to add MMU mapping at here.

Reply:
I do not have the bandwidth to modify flexcan driver for now... Could we left it as-is?

Reply:
NP, we can do it late.

Reply:
then I'll mark this resolved